views/web.py

```diff
@@ -286,6 +280,7 @@ def love():
             flash('Secret love sent to {}!'.format(real_display_str))
             return redirect(url_for('home'))
         else:
+            flash('Love sent to {}!'.format(real_display_str))
```

Comment:
I think I've misunderstood @rockdog's comment above  :"I think there is a redirect + message missing here.". Removing the flash message.